cypress/fixtures/search-view-shows-no-results.json

```diff
@@ -0,0 +1,5 @@
+{
+    "data": {
```

Comment:
Nice job with making a Fixture for no shows so that you can tests a variety of sad paths.